web/HW.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+
+<head>
+    <meta charset="utf-8" />
+
+
+    <link href="Css/Hw.css" rel="stylesheet" type="text/css" media="screen" />
+
+</head>
+
+<h3> Welcome to Lama alaskar Page .</h3>
+
+<body>
+
+    <h4>Nice to see you here I hope you get ready to mypage trip</h4>
+    <div id="container"> </div>
+    <p id="logo">
+        <a href="#"><img src="Unknown.jpeg"> </a>
+    </p>
+
+
+    <h3>About Lama:</h3>
+    <p id="Who">Lama alaskar Fresh graduated Student from King saud University at the second of januray at 2020.I belive that Computer science can change hall life by making the idea gets true and apply it but it all about understanding what goiong on Also I'm workig
+        to be a great developer and work with great Software Engineering team .
+
+    </p>
+    <h3>About Training Program:</h3>
+    <p id="train">The training program was in a deanship of Library affaires at King saud University it requires from us as a team to design and implment Booking locker System that let student booking from it so we design the websites using Html and css for the front
+        end and php for the back end and database part also We use mySql to design the database on the system .</p>
+
+    <h4>About SEI Program:</h4>
+
+    <p id="Soft">The Software Engineering Immersive is a demanding coding bootcamp that embodies the rigor and challenges developers face each day on the job. We believe that, with hard work and the help of our expert team, anyone can gain the skills to launch a successful
```

Comment:
be careful with naming conventions as you may lose ID names for your elements keep them consistent